src/wolfkrow/core/tasks/nuke_render.py

```diff
@@ -29,14 +29,35 @@ def export_to_command_line(self, job_name, temp_dir=None, deadline=False):
             export_json=export_json
         )
 
-        updated_exported = []
         # Append a "$" to the end of the command so that nuke does not consume 
         # the last argument as a frame number/range.
         for export in exported:
-            obj, command = export
-            command = "{} $".format(command)
-            updated_exported.append((obj, command))
-        return updated_exported
+            args = "{} $".format(export.task_args)
+            export.task_args = args
+        return exported
+
+    def _command_line_sanitize_attribute(
+        self, attribute_name, attribute_value, deadline=False
+    ):
+        """
+        Processes the attribute value to prepare it for use on the command line.
```

Comment:
This isn't true... Perhaps we should rename this method to just `sanitize_attribute`. And it gives every task the ability to do additional processing on it's TaskAttribute's.

Reply:
Especially since the command_line export method is deprecated now.